integration_tests/features/polar2grid.feature

```diff
@@ -21,12 +21,12 @@ Feature: Test polar2grid output images
     | polar2grid.sh avhrr gtiff -vv -f | avhrr/input/test1 | avhrr/output/test1 |
 
   Examples: MODIS
-    | command                                                                                                                                             | source            | output             |
-    | polar2grid.sh crefl gtiff -vv --true-color --false-color --fornav-D 10 --grid-configs ${POLAR2GRID_HOME}/grid_configs/grid_example.conf -g miami -f | modis/input/test1 | modis/output/test1 |
+    | command                                                                                                                                      | source            | output             |
+    | polar2grid.sh crefl gtiff -vv --true-color --false-color --fornav-D 10 --grid-configs ${DATAPATH}/grid_configs/grid_example.conf -g miami -f | modis/input/test1 | modis/output/test1 |
```

Comment:
DATAPATH seems like an odd name for this. I would assume DATAPATH is where the test data and expected data are located. This seems like it is where the tarball/bundle is, right?

Reply:
DATAPATH is where the test data is stored, yes. the config files are with the rest of the test data (/data/test_data).

Reply:
Ooohhh ok. Sounds good.